tests/OSPSuite.Core.Tests/Batch/SimulationResultsExporterSpecs.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Threading.Tasks;
+using FakeItEasy;
+using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Batch;
+using OSPSuite.Core.Batch.Mappers;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.Mappers;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Core.Services;
+using OSPSuite.Infrastructure.Services;
+using OSPSuite.Utility;
+
+namespace PKSim.BatchTool
+{
+   public abstract class concern_for_SimulationResultsExporter : ContextSpecification<ISimulationResultsExporter>
+   {
+      protected IDataRepositoryTask _dataRepositoryTask;
+      private IQuantityPathToQuantityDisplayPathMapper _quantityDisplayPathMapper;
+      protected ISimulationResultsToBatchSimulationExportMapper _batchSimulationExportMapper;
+      protected ISimulation _simulation;
+      protected DataRepository _results;
+      protected string _fileName;
+
+      protected override void Context()
+      {
+         _dataRepositoryTask = A.Fake<IDataRepositoryTask>();
+         _quantityDisplayPathMapper = A.Fake<IQuantityPathToQuantityDisplayPathMapper>();
+         _batchSimulationExportMapper = A.Fake<ISimulationResultsToBatchSimulationExportMapper>();
+         sut = new SimulationResultsExporter(_dataRepositoryTask, _quantityDisplayPathMapper, _batchSimulationExportMapper);
+
+         _simulation = A.Fake<ISimulation>();
+         _results = new DataRepository();
+         _fileName = FileHelper.GenerateTemporaryFileName();
+      }
+
+      public override void GlobalCleanup()
+      {
+         base.GlobalCleanup();
+         FileHelper.DeleteFile(_fileName);
+      }
+   }
+
+   public class When_exporting_simulation_results_to_csv : concern_for_SimulationResultsExporter
+   {
+      private List<DataTable> _dataTables;
+
+      protected override void Context()
+      {
+         base.Context();
+         var dataTable = new DataTable("TEST");
+
+         _dataTables = new List<DataTable> {dataTable};
+         A.CallTo(_dataRepositoryTask).WithReturnType<IEnumerable<DataTable>>().Returns(_dataTables);
+      }
+
+      [Observation]
+      public async Task should_create_a_data_table_with_all_results_from_the_simulation_and_export_it_to_the_file()
```

Comment:
Checking the appveyor logs here. This test appears gren in the list, but without a time.